sklearn/feature_extraction/text.py

```diff
@@ -1145,6 +1173,10 @@ class TfidfVectorizer(CountVectorizer):
         preprocessing and n-grams generation steps.
         Only applies if ``analyzer == 'word'``.
 
+    token_processor: callable or None (default)
+        Override the token processing step while preserving the
```

Comment:
Needs to be clear that it applies to a sequence of tokens, rather than each token individually.